src/library/dao/trackschema.h

```diff
@@ -54,4 +55,15 @@ const QString TRACKLOCATIONSTABLE_NEEDSVERIFICATION = "needs_verification";
 
 const QString REKORDBOX_ANALYZE_PATH = "analyze_path";
 
+// Simple static-only class for getting the ordered subset of columns used in the track headers.
+class TrackSchema {
+public:
+    // GetColumns returns a QStringList of all the relevant columns that can be displayed to the
+    // user.
+    static QStringList GetColumnNames();
```

Comment:
A static constant should work

Reply:
> Just wondering: Does this make it possible to modify the default column order? I'ld love to move the color column to the left by default for 2.3 if possible.

yes, because headers are saved and restored by name, the default ordering is irrelevant.  I think this code is working in 2.2 so this shouldn't break people not on the beta build.  Unfortunately 2.3 has been saving header states with blank names, so column loading will probably break for people.